packages/vite-plugin-rune/test/fixtures/outer.ts

```diff
@@ -0,0 +1,3 @@
+export function outer() {
```

Comment:
FFTI: Not sure inner and outer are the best names, but can't think of anything better!